installation.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: home
+title: Installation
+nav_order: 2
+nav_titles: true
+titles_max_depth: 2
+---
+## Enabeling on linux systems
+run the following command:
+```bash
+sysctl net.mptcp.enabled=1
+```
+
+## Force MPTCP
+apps can be forces to use one of the following methods
+
+- [mptcpize](https://www.mankier.com/8/mptcpize)
+
+    `mptcpize run <app command>`  
+    this works by modifying the behaviour of the underlaying lib-c  
+    *note: this does does not always work and should not be used for production*
+
+- [GODEBUG](https://go-review.googlesource.com/c/go/+/507375)
+
+    whith GO, the lib-c is not used, so `mptcpize` will not work.
+    to force MPTCP the environment variable `GODEBUG=multipathtcp=1` can be used.
+
+- [eBPF](https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git/commit/?id=ddba122428a7)
+
+- [systemtap](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/getting-started-with-multipath-tcp_configuring-and-managing-networking#preparing-rhel-to-enable-mptcp-support_getting-started-with-multipath-tcp)
+
+## ss commands
+the `ss` command on linux systems has an option to list MPTCP socket `-M`. the recommanded call is `ss -Menita`
+
+- `M`, MPTCP sockets
+- `t`, TCP including subflows created by MPTCP
+- `n`, prevent the port number to protocol vonversion
+- `i`, info on the connection TCP_INFO, MPTCP_INFO, etc.
+- `e`, more info
+- `a`, dysplay all socket not just "established" ones
+
+[man page](https://www.commandlinux.com/man-page/man8/ss.8.html)
+
+
+## Making use of multiple streams
```

Comment:
streams? do you mean subflows or paths? We should probably use `subflows` but explain it before